std_msgs/Header header

geometry_msgs/Pose car_position_estimation

int32[] detections_assignment

float64[] time_spent

float64 cost

bool recovery_applied